apps/src/code-studio/videos.js

```diff
@@ -515,7 +516,26 @@ function addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight) {
     }
   );
 
-  videoPlayer.on('ended', onVideoEnded);
+  const analyticsData = {
+    url: location.href,
+    video: videoInfo.download,
+    fallback: 'code.org',
```

Comment:
Is `fallback: 'code.org'` our indication that we used the fallback player? Are there other values `fallback` could have?

Reply:
Correct, just an explicit note that it's using our locally hosted video.

Presumably we'd also have a `fallback : youtube-blocked` or the equivalent for the middle case, once we're able to track those events on it. Or, alternatively, we may just decide that just knowing the video url is sufficient. Def something that we can iterate on.